components/EmptyState/EmptyState.js

```diff
@@ -1,27 +1,27 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-
+import React from "react"
+import PropTypes from "prop-types"
 
 class EmptyState extends React.PureComponent {
-
   render() {
+    const { title, message, children } = this.props
```

Comment:
This PR removes a lot of semicolons on line endings. At least personally I must say this is highly confusing, as in JS in general, statements have to be terminated by a semicolon. Does eslint enforce that with this configuration? [cockpit's](https://github.com/cockpit-project/cockpit/blob/master/.eslintrc.json) doesn't, and we don't have anything special there.

This should maybe get agreed upon between the welder-web developers  first, as it's a lot of noise.

Reply:
I'm not familiar with what's required vs what's standard. I'd defer to whatever the cockpit devs prefer or would recommend. 

That said, seeing it like this is not what I'm use to. If there are some cases where semicolons are required and others where they're optional, it might take me some time to fully grasp those rules.